src/webnn_native/dml/GraphDML.h

```diff
@@ -47,6 +46,24 @@
 #include "webnn_native/ops/Transpose.h"
 #include "webnn_native/ops/Unary.h"
 
+#if defined(WEBNN_ENABLE_GPU_BUFFER)
+// Support DirectMLX
+#    include <initguid.h>
+#    include <wrl/client.h>
+#    include <wrl/implements.h>
+
+#    include <Windows.h>
+#    include <d3d12.h>
+
+#    define DML_TARGET_VERSION_USE_LATEST 1
+#    include <DirectML.h>
+#    include "DirectMLX.h"
+
+#    include "dawn/native/dml/deps/src/dmldevice.h"
```

Comment:
Where is this file hosted? Is it added into the DEPS?

Reply:
It hosts in Dawn repo seeing https://github.com/fujunwei/dawn/commits/webnn_webgpu_interop which need to be refactored.